core/src/main/scala/cromwell/util/PromiseActor.scala

```diff
@@ -42,7 +42,7 @@ object PromiseActor {
     promise.future
   }
 
-  def props(promise: Promise[Any], sendTo: ActorRef, msg: Any): Props = Props(new PromiseActor(promise, sendTo, msg))
+  def props(promise: Promise[Any], sendTo: ActorRef, msg: Any): Props = Props(new PromiseActor(promise, sendTo, msg)).withDispatcher(EngineDispatcher)
```

Comment:
This one wasn't clear to me as to which dispatcher to use. It's not in `engine`, but it's fully self contained in here and this tidbit is only called in `engine`.